src/main/java/eu/dissco/nusearch/property/RabbitMQProperties.java

```diff
@@ -0,0 +1,33 @@
+package eu.dissco.nusearch.property;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@Validated
+@ConfigurationProperties(prefix = "rabbitmq")
+public class RabbitMQProperties {
+
```

Comment:
Do we want these to be configurable? they seem static to me. 
Also, I think we should document these exchanges and connections

Reply:
This sets default values which can be overwritten with the application properties. So this is the default but if we change something we can put that change in the env's